src/components/listings.jsx

```diff
@@ -72,7 +95,15 @@ export default class Listings extends Component {
               </thead>
 
               <tbody>
-                {this.state.listings.map(listing => (
+                <div>
```

Comment:
You can't put a \<div\> inside of a \<body\>. Only cells can be inside of the table, so you can have \<div\> inside of cells but not directly inside of the table.